src/main/java/com/booksaw/betterTeams/util/FoliaUtils.java

```diff
@@ -0,0 +1,56 @@
+package com.booksaw.betterTeams.util;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+
+import static com.booksaw.betterTeams.Main.plugin;
+
+public class FoliaUtils {
```

Comment:
Could you add some javadocs here explaining the basics of folia and its current limitations (can just link to existing documentation) so people have a point of reference